elasticassignment/ageofempires/management/commands/index_technologies.py

```diff
@@ -0,0 +1,46 @@
+from django.core.management.base import BaseCommand
+from elasticsearch_dsl.connections import connections
+from elasticsearch.helpers import bulk
+import requests
+import elasticsearch
+
+class Command(BaseCommand):
+
+    def handle(self, *args, **kwargs):
+        config = {
+            'host': '127.0.0.1'
```

Comment:
what is the purpose of `ELASTICSEARCH_DSL` setting your created in `settings.py` if you have to hard code the host here!!